index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="reset" href="./assets/css/reset.css"/>
```

Comment:
Are we going to use the reset from class or use normalize css.

Reply:
Go big or go home! Lets use normalize

Reply:
Sounds good lets use normalize.